airmozilla/main/static/main/js/eventstatus.js

```diff
@@ -0,0 +1,19 @@
+(function () {
```

Comment:
Use the jQuery onload event handler to be safe. That way it doesn't execute until the `document.onload` really has fired. (as opposed to loading when the javascript file has been loaded). 
The way things work today in airmo, this is basically the same thing but it's safer to be safe. 

So change this line to `$(function() {` and change the very last line to `});`